test/language/module-code/instn-star-props-nrml-1_FIXTURE.js

```diff
@@ -22,3 +22,5 @@ export { indirectIdName } from './instn-star-props-nrml-indirect_FIXTURE.js';
 export { indirectIdName as indirectIdName2 } from './instn-star-props-nrml-indirect_FIXTURE.js';
 
 export * from './instn-star-props-nrml-star_FIXTURE.js';
+
+export * as exportns from './instn-star-props-nrml-star-as-exportns_FIXTURE.js';
```

Comment:
I assume you will still split this part into a new file?

same for `test/language/module-code/instn-star-props-nrml.js`

In this case, tests LGTM.

Reply:
I'm actually really not sure what you are asking, @leobalter